source/views/more/index.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react'
+import {SectionList, StyleSheet} from 'react-native'
+
+import * as c from '@frogpond/colors'
+import {useDebounce} from '@frogpond/use-debounce'
+import {LoadingView, NoticeView} from '@frogpond/notice'
+import {openUrl} from '@frogpond/open-url'
+import {Row} from '@frogpond/layout'
+import {
+	ListSeparator,
+	ListSectionHeader,
+	largeListProps,
+	Title,
+	ListRow,
+	emptyList,
+} from '@frogpond/lists'
+import {SearchData, LinkGroup, LinkResults, LinkValue} from './types'
+
+import {useFetch} from 'react-async'
+import {NativeStackNavigationOptions} from '@react-navigation/native-stack'
+import {useNavigation} from '@react-navigation/native'
+import {ChangeTextEvent} from '../../navigation/types'
+import deburr from 'lodash/deburr'
+import words from 'lodash/words'
+import groupBy from 'lodash/groupBy'
+
+const styles = StyleSheet.create({
+	wrapper: {
+		flex: 1,
+	},
+	row: {
+		marginVertical: 5,
+	},
+	contentContainer: {
+		flexGrow: 1,
+	},
+})
+
+function splitToArray(str: string) {
+	return words(deburr(str.toLowerCase()))
+}
+
+function linkToArray(data: LinkValue) {
+	return Array.from(new Set([...splitToArray(data.label)]))
+}
+
+let groupResults = (
+	searchQuery: string,
+	rawData: LinkResults[],
+	groupedOriginal: LinkGroup[],
+) => {
+	if (!rawData) {
+		return emptyList
+	}
+
+	if (searchQuery.length < 3) {
+		return groupedOriginal
+	}
+
+	let filtered = rawData.flatMap((data) =>
+		data.values.filter((value) =>
+			linkToArray(value).some((value) => value.includes(searchQuery)),
+		),
+	)
+
+	let groupedResults = groupBy(filtered, (result) => result.label[0] || '?')
+
+	let groupedLinks = Object.entries(groupedResults).map(([key, value]) => ({
+		title: key,
+		data: value,
+	}))
+
+	return groupedLinks
+}
+
+const useSearchLinks = () => {
+	const url = 'https://wp.stolaf.edu/wp-json/site-data/sidebar/a-z'
+	return useFetch<SearchData>(url, {
+		headers: {accept: 'application/json'},
+	})
+}
+
+function MoreView(): JSX.Element {
+	let navigation = useNavigation()
+
+	let [query, setQuery] = React.useState('')
+	let searchQuery = useDebounce(query.toLowerCase(), 200)
+
+	let {
+		data: {az_nav: {menu_items: menuItems = []} = {}} = {},
```

Comment:
```suggestion
		data,
```

Then later

```ts 
let menuItems = data?.az_nav?.menu_items ?? []
```

Reply:
Great idea. This comment inspired me to break this up into two parts:
- `data = []` lives at #6206 
- `data?.az_nav?.menu_items` lives at https://github.com/frog-pond/ccc-server/pull/617